gpu_bdb/bdb_tools/cluster_startup.py

```diff
@@ -35,8 +35,25 @@ def attach_to_cluster(config, create_sql_context=False):
     scheduler_file = config.get("scheduler_file_path")
     host = config.get("cluster_host")
     port = config.get("cluster_port", "8786")
+    start_local_cluster = config.get("start_local_cluster", False)
+
+    if start_local_cluster:
+        from dask_cuda import LocalCUDACluster
+        cluster = LocalCUDACluster(
+            n_workers=int(os.environ.get("NUM_WORKERS", 16)),
+            device_memory_limit=os.environ.get("DEVICE_MEMORY_LIMIT", "15GB"),
```

Comment:
This seems really low to me. Any reason we set it to `29GB` for `30GB` cards.

Reply:
Picked the `DEVICE_MEMORY_LIMIT` default value from `cluster-startup.sh`, which is `15GB`.

Reply:
@ayushdg / @randerzander/ @ChrisJar   , Any reason we have the default set this low ?

Reply:
I think `20GB` should be fair. I've seen OOM's happen at 25GB with the dask-sql queries which we can investigate further.

Reply:
Changed the default values to `20GB` at both places.